features/braille/brailleSelector.js

```diff
@@ -61,6 +66,24 @@ export const getAllResults = createSelector([getInput], (input) => {
   return decodedVariants
 })
 
-export const getInputSolution = createSelector([getInput], (input) => {
-  return input.map((entry) => toUtf(entry), '').join('')
+export const getIsVariantSelected = createSelector([], () => {
+  return false
+})
+
+const inputToDecodeItems = (input) => {
```

Comment:
I prefer to name and place this method to the corresponding place as it in morse.